lib/index.ts

```diff
@@ -23,16 +26,55 @@ function inspect(root: string, targetFile?: string, options?: any) {
         dockerImageId: result[1].imageId,
       };
       const pkg: any = result[1].package;
+      const dockerfilePackages = result[2]
+        ? getDockerfileDependencies(result[2].dockerfilePackages,
+                                    pkg.dependencies)
+        : [];
+
       pkg.docker = pkg.docker || {};
-      pkg.docker.baseImage = result[2];
       pkg.docker.binaries = result[1].binaries;
+      pkg.docker = {
+        ...pkg.docker,
+        ...result[2],
+        dockerfilePackages,
+      };
+
       return {
         plugin: metadata,
         package: pkg,
       };
     });
 }
 
+function getDockerfileDependencies(dockerfilePackages: DockerFilePackages,
```

Comment:
This function is more of `adding` the docker file package dependencies if I'm understanding it correctly.
Maybe `addDockerfilePackagesDependencies` ?

Maybe a small comment here explaining why we do this would be nice :)

Reply:
Done